lib/filter.js

```diff
@@ -12,13 +12,35 @@ var TweetFilter = function(filters_or_dirpath, excluded_terms) {
 
 // load all files from a directory and use their exports as filters
 // the files should export arrays of strings/regex expressions
-TweetFilter.getFiltersFromDirectory = function(dir) {
+TweetFilter.getFiltersFromDirectory = function(dir, filters) {
     dir = path.resolve(dir);
     var filter_files = fs.readdirSync(dir);
     var joined_files = [];
-    filter_files.forEach(function(file) {
-        joined_files.push(path.join(dir, file));
-    })
+
+    if( filters.length == 0 ) {
```

Comment:
This should be `if ( !filters || filters.length == 0 ) {` to allow for a null filters being passed in.